The error is:

```
2026-02-04T03:52:02.4803162Z --
2026-02-04T03:52:02.4803445Z Exit Code: 1
2026-02-04T03:52:02.4803612Z 
2026-02-04T03:52:02.4803737Z Command Output (stdout):
2026-02-04T03:52:02.4804119Z --
2026-02-04T03:52:02.4804776Z lldb version 23.0.0git (https://github.com/llvm/llvm-project revision fc6eac533cd08a4424bdedea36970be393557550)
2026-02-04T03:52:02.4805715Z   clang revision fc6eac533cd08a4424bdedea36970be393557550
2026-02-04T03:52:02.4806330Z   llvm revision fc6eac533cd08a4424bdedea36970be393557550
2026-02-04T03:52:02.4807114Z Skipping the following test categories: msvcstl, dsym, pdb, gmodules, debugserver, objc
2026-02-04T03:52:02.4807652Z 
2026-02-04T03:52:02.4807759Z --
2026-02-04T03:52:02.4808029Z Command Output (stderr):
2026-02-04T03:52:02.4808370Z --
2026-02-04T03:52:02.4810291Z UNSUPPORTED: LLDB (/home/gha/actions-runner/_work/llvm-project/llvm-project/build/bootstrapping-build/bin/clang-x86_64) :: test_dsym (TestIteratorFromStdModule.TestCase) (test case does not fall in any category of interest for this run) 
2026-02-04T03:52:02.4812702Z FAIL: LLDB (/home/gha/actions-runner/_work/llvm-project/llvm-project/build/bootstrapping-build/bin/clang-x86_64) :: test_dwarf (TestIteratorFromStdModule.TestCase)
2026-02-04T03:52:02.4814759Z FAIL: LLDB (/home/gha/actions-runner/_work/llvm-project/llvm-project/build/bootstrapping-build/bin/clang-x86_64) :: test_dwo (TestIteratorFromStdModule.TestCase)
2026-02-04T03:52:02.4816318Z ======================================================================
2026-02-04T03:52:02.4816956Z FAIL: test_dwarf (TestIteratorFromStdModule.TestCase)
2026-02-04T03:52:02.4817569Z ----------------------------------------------------------------------
2026-02-04T03:52:02.4818101Z Traceback (most recent call last):
2026-02-04T03:52:02.4819071Z   File "/home/gha/actions-runner/_work/llvm-project/llvm-project/lldb/packages/Python/lldbsuite/test/lldbtest.py", line 1844, in test_method
2026-02-04T03:52:02.4820400Z     return attrvalue(self)
2026-02-04T03:52:02.4821422Z   File "/home/gha/actions-runner/_work/llvm-project/llvm-project/lldb/packages/Python/lldbsuite/test/decorators.py", line 160, in wrapper
2026-02-04T03:52:02.4822450Z     return func(*args, **kwargs)
2026-02-04T03:52:02.4823365Z   File "/home/gha/actions-runner/_work/llvm-project/llvm-project/lldb/packages/Python/lldbsuite/test/decorators.py", line 160, in wrapper
2026-02-04T03:52:02.4824450Z     return func(*args, **kwargs)
2026-02-04T03:52:02.4825688Z   File "/home/gha/actions-runner/_work/llvm-project/llvm-project/lldb/test/API/commands/expression/import-std-module/iterator/TestIteratorFromStdModule.py", line 30, in test
2026-02-04T03:52:02.4826998Z     self.expect("expr move_begin++")
2026-02-04T03:52:02.4827991Z   File "/home/gha/actions-runner/_work/llvm-project/llvm-project/lldb/packages/Python/lldbsuite/test/lldbtest.py", line 2463, in expect
2026-02-04T03:52:02.4828959Z     self.runCmd(
2026-02-04T03:52:02.4830333Z   File "/home/gha/actions-runner/_work/llvm-project/llvm-project/lldb/packages/Python/lldbsuite/test/lldbtest.py", line 1041, in runCmd
2026-02-04T03:52:02.4831482Z     self.assertTrue(self.res.Succeeded(), msg + output)
2026-02-04T03:52:02.4832289Z AssertionError: False is not true : Command 'expr move_begin++' did not return successfully
2026-02-04T03:52:02.4832981Z Error output:
2026-02-04T03:52:02.4833430Z error: Couldn't materialize: Invalid type: Cannot determine size
2026-02-04T03:52:02.4834237Z error: errored out in DoExecute, couldn't PrepareToExecuteJITExpression
2026-02-04T03:52:02.4834679Z 
2026-02-04T03:52:02.4835393Z Config=x86_64-/home/gha/actions-runner/_work/llvm-project/llvm-project/build/bootstrapping-build/bin/clang
2026-02-04T03:52:02.4836288Z ======================================================================
2026-02-04T03:52:02.4836859Z FAIL: test_dwo (TestIteratorFromStdModule.TestCase)
2026-02-04T03:52:02.4837478Z ----------------------------------------------------------------------
2026-02-04T03:52:02.4838035Z Traceback (most recent call last):
2026-02-04T03:52:02.4839476Z   File "/home/gha/actions-runner/_work/llvm-project/llvm-project/lldb/packages/Python/lldbsuite/test/lldbtest.py", line 1844, in test_method
2026-02-04T03:52:02.4840553Z     return attrvalue(self)
2026-02-04T03:52:02.4841547Z   File "/home/gha/actions-runner/_work/llvm-project/llvm-project/lldb/packages/Python/lldbsuite/test/decorators.py", line 160, in wrapper
2026-02-04T03:52:02.4842590Z     return func(*args, **kwargs)
2026-02-04T03:52:02.4843580Z   File "/home/gha/actions-runner/_work/llvm-project/llvm-project/lldb/packages/Python/lldbsuite/test/decorators.py", line 160, in wrapper
2026-02-04T03:52:02.4844729Z     return func(*args, **kwargs)
2026-02-04T03:52:02.4845953Z   File "/home/gha/actions-runner/_work/llvm-project/llvm-project/lldb/test/API/commands/expression/import-std-module/iterator/TestIteratorFromStdModule.py", line 30, in test
2026-02-04T03:52:02.4847274Z     self.expect("expr move_begin++")
2026-02-04T03:52:02.4848312Z   File "/home/gha/actions-runner/_work/llvm-project/llvm-project/lldb/packages/Python/lldbsuite/test/lldbtest.py", line 2463, in expect
2026-02-04T03:52:02.4849518Z     self.runCmd(
2026-02-04T03:52:02.4850368Z   File "/home/gha/actions-runner/_work/llvm-project/llvm-project/lldb/packages/Python/lldbsuite/test/lldbtest.py", line 1041, in runCmd
2026-02-04T03:52:02.4851391Z     self.assertTrue(self.res.Succeeded(), msg + output)
2026-02-04T03:52:02.4852337Z AssertionError: False is not true : Command 'expr move_begin++' did not return successfully
2026-02-04T03:52:02.4853019Z Error output:
2026-02-04T03:52:02.4853454Z error: Couldn't materialize: Invalid type: Cannot determine size
2026-02-04T03:52:02.4854245Z error: errored out in DoExecute, couldn't PrepareToExecuteJITExpression
2026-02-04T03:52:02.4854722Z 
2026-02-04T03:52:02.4855264Z Config=x86_64-/home/gha/actions-runner/_work/llvm-project/llvm-project/build/bootstrapping-build/bin/clang
2026-02-04T03:52:02.4856226Z ----------------------------------------------------------------------
2026-02-04T03:52:02.4856786Z Ran 3 tests in 6.703s
2026-02-04T03:52:02.4856997Z 
2026-02-04T03:52:02.4857134Z FAILED (failures=2, skipped=1)
2026-02-04T03:52:02.4857492Z Failures per category:
2026-02-04T03:52:02.4857808Z libc++ - 2
2026-02-04T03:52:02.4858053Z dwarf - 1
2026-02-04T03:52:02.4858304Z expression - 2
2026-02-04T03:52:02.4858647Z std-module - 2
2026-02-04T03:52:02.4858930Z dwo - 1
2026-02-04T03:52:02.4859074Z 
2026-02-04T03:52:02.4859184Z --
2026-02-04T03:52:02.4859545Z 
2026-02-04T03:52:02.4859651Z ********************
2026-02-04T03:52:25.3238192Z Testing:  0.. 10.. 20.. 30.. 40.. 50.. 60.. 70.. 80.. 90.. 
2026-02-04T03:52:25.3281618Z ********************
2026-02-04T03:52:25.3281949Z Failed Tests (1):
2026-02-04T03:52:25.3282601Z   lldb-api :: commands/expression/import-std-module/iterator/TestIteratorFromStdModule.py
2026-02-04T03:52:25.3283162Z 
```

@michael137 It seems that LLDB is choking on the evaluation of `it++` after the patch.

--- a/lldb/test/API/commands/expression/import-std-module/iterator/TestIteratorFromStdModule.py
+++ b/lldb/test/API/commands/expression/import-std-module/iterator/TestIteratorFromStdModule.py
@@ -9,8 +9,7 @@ from lldbsuite.test import lldbutil
 
 class TestCase(TestBase):
     @add_test_categories(["libc++"])
-    @skipIf(compiler=no_match("clang"))
-    @skipIf(macos_version=["<", "15.0"])
+    @skipIf
     def test(self):
         self.build()
 

--- a/lldb/test/API/commands/expression/import-std-module/iterator/TestIteratorFromStdModule.py
+++ b/lldb/test/API/commands/expression/import-std-module/iterator/TestIteratorFromStdModule.py
@@ -9,8 +9,9 @@ from lldbsuite.test import lldbutil
 
 class TestCase(TestBase):
     @add_test_categories(["libc++"])
-    @skipIf
-    def test(self):
+    @skipIf(compiler=no_match("clang"))
+    @expectedFailureAll(bugnumber="https://github.com/llvm/llvm-project/issues/149477")
+    def test_xfail(self):
         self.build()
 
         lldbutil.run_to_source_breakpoint(
@@ -18,13 +19,6 @@ class TestCase(TestBase):
         )
 
         self.runCmd("settings set target.import-std-module true")
-
-        iter_type = "std::move_iterator<std::__wrap_iter<int *> >"
-
-        self.expect_expr("move_begin", result_type=iter_type)
-        self.expect_expr("move_begin[0]", result_type="int", result_value="1")
-
-        self.expect_expr("move_begin + 3 == move_end", result_value="true")
 
         self.expect("expr move_begin++")
         self.expect_expr("move_begin + 2 == move_end", result_value="true")

